src/theme.ts

```diff
@@ -49,8 +49,9 @@ export const lightTheme = createTheme({
         colorInherit: {
           backgroundColor: "#FFFFE9",
           color: "#141414",
-          border: "1px solid #FFFFE9",
           boxShadow: "transparent",
+          backgroundImage: "none !important",
+          borderColor: "transparent",
```

Comment:
What are these two rules doing to the styling? Why are they necessary?

Reply:
backgroundImage:'none !important" is not necessary because I added this on the MUIPaper.
But borderColor:'transparent' is necessary because MUIAppBar is having paper and the border will be appear which I gave on the MUIPaper on theme